src/tdastro/math_nodes/basic_math_node.py

```diff
@@ -0,0 +1,185 @@
+"""Nodes that perform basic math operations that can be specified as strings.
+
+The goal of this library is to save users from needing to create a bunch of
+small FunctionNodes to perform basic math.
+"""
+
+import ast
+
+# Disable unused import because we need all of these imported
+# so they can be used during evaluation of the node.
+import math  # noqa: F401
+
+import jax.numpy as jnp  # noqa: F401
+import numpy as np  # noqa: F401
+
+from tdastro.base_models import FunctionNode
+
+
+class BasicMathNode(FunctionNode):
```

Comment:
Totally irrelevant to this PR, but as a grad student I've definitely wished for this sort of conversion as a callable library, especially if C stdlib was one of the "basic math" languages that could be converted.